Frontend/ui-customer/src/Services/Backend.purs

```diff
@@ -331,24 +331,26 @@ rideSearchBT payload = do
 
 
 makeRideSearchReq :: Number -> Number -> Number -> Number -> Address -> Address -> SearchReq
-makeRideSearchReq slat slong dlat dlong srcAdd desAdd=
-     SearchReq { "contents" : OneWaySearchReq{
-                                               "destination" : SearchReqLocation {
-                                                        "gps" : LatLong {
-                                                            "lat" : dlat ,
-                                                            "lon" : dlong
-                                                            },
-                                                        "address" : (LocationAddress desAdd)
-                                               },
-                                               "origin" : SearchReqLocation {
-                                                "gps" : LatLong {
-                                                            "lat" : slat ,
-                                                            "lon" : slong
-                                                },"address" : (LocationAddress srcAdd)
-                                               }
-                                              },
-                 "fareProductType" : "ONE_WAY"
-                }
+makeRideSearchReq slat slong dlat dlong srcAdd desAdd =
+    let appConfig = CP.getAppConfig CONST.appConfig
```

Comment:
`import Constants as CONST` is not required. it can be imported from `ConfigProvider` itself